src/data/pokemon/species_info.h

```diff
@@ -36,6 +36,7 @@
         .abilities = {ABILITY_STATIC, ABILITY_NONE, ABILITY_LIGHTNING_ROD},\
         .bodyColor = BODY_COLOR_YELLOW,                                    \
         .noFlip = flip,                                                    \
+        .flags = SPECIES_FLAG_GENDER_DIFFERENCE,                           \
```

Comment:
Adding this in the macro will cause *all* Pikachu forms to be counted as having gender differences, which is not the case.